apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -190,7 +193,12 @@ def render_controls(template, elem):
                 template,
                 {
                     "render_button": True,
-                    "button_class": elem.attrib.get("type", ""),
+                    "button_class": type_name,
+                    "button_label": elements.element_labels.get(type_name)
+                    or additional_button_labels.get(
+                        type_name,
+                        type_name,
+                    ),
```

Comment:
I believe this needs to take element extensions into account.

Reply:
So it will fallback to the element type name if it doesn't exist. The alternative is to expect that `pl-drawing` extension elements define e.g. a `name` on `class CustomElement(BaseElement)`. I don't think many people will do that.

Reply:
If an element supports extensions (I think `pl-drawing` is actually the only one that does), any first-party element features should be possible to use with extensions too. Given that, I think a better API would be to have the first-party elements have a `label` that's read, with the element name as a fallback. Then extensions work exactly the same as first-party elements.

Reply:
If so, that's a lift I'm not super enthusiastic about doing, especially given the amount of other times I have passed over linting or fixing this code since it most likely needs a full rewrite.

Reply:
Let's back up a second here: grep for `get_button_tooltip` in `apps/prairielearn/elements/pl-drawing`. All elements already have a defined title; they just aren't visible to the accessibility checker since it's done client-side in JavaScript.

Reply:
Ah man. Didn't see that.

Reply:
Changed to a dummy label in 7445e82f7cb6c569b83443ee2d5cb35a749a8f3c to satisfy the checker.